php/class-customize-snapshot-manager.php

```diff
@@ -747,6 +748,23 @@ public function enqueue_frontend_scripts() {
 	}
 
 	/**
+	 * Enqueue admin scripts.
+	 *
+	 * These files control the behavior and styling of links to remove settings.
+	 * Published snapshots can't be edited, so these files are not needed on those pages.
+	 *
+	 * @param String $hook Current page.
+	 */
+	public function enqueue_admin_scripts( $hook ) {
+		global $post;
+		$handle = 'customize-snapshots-admin';
```

Comment:
This could be moved into the conditional, and add return space after the global.